.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
do we still need this file 🤔? I would suggest we include all sorts of settings in `.eslintrc.json`. @reiosantos

Reply:
This file has been remnoved.